esp_encrypted_img/test/test.c

```diff
@@ -304,3 +306,20 @@ TEST_CASE("Sending random size data at once", "[encrypted_img]")
     }
     free(args);
 }
+
+TEST_CASE("Test canceling decryption frees memory", "[encrypted_img]")
+{
+    esp_decrypt_cfg_t cfg = {
+        .rsa_pub_key = (char *)rsa_private_pem_start,
+        .rsa_pub_key_len = rsa_private_pem_end - rsa_private_pem_start,
+    };
+    int free_bytes_start = xPortGetFreeHeapSize();
```

Comment:
Just note: there is also `[leaks]` tag that we can in `TEST_CASE` declaration. We will improve this further, for now we can accept current solution.